app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -375,6 +382,7 @@ export default {
     clearMessage() {
       this.message = '';
       this.attachedFiles = [];
+      this.clearMails = true;
```

Comment:
Once we set `clearMails` to **_true_**, it is always **_true_** until the page is refreshed. So, if the next time a ccEmail is added, it will be cleared immidiately.

Reply:
@fayazara Handled it [here](https://github.com/chatwoot/chatwoot/pull/3098/files#diff-4035b7b3fa6e8a2258fcf57b6e68d6476b5a96b2c368fe299ea2f42603f637a4R363)